### PR TITLE
PP-2434 add cookie messaging

### PR DESCRIPTION
**Pull request by jonheslop**

### CODE REVIEW

File: source/cookies.html.erb [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+        <li>remember you once you've signed in</li>
+        <li>remember the notifications you've seen so we don't show them again</li>
+        <li>measure how you use the website so it can be updated and improved based on your needs</li>
+      </ul>
+
+      <p>You’ll normally see a message on the site before we store a cookie on your computer.</p>
+
+      <p>Find out <a href="http://www.aboutcookies.org/">how to manage cookies</a>.</p>
+
+      <h2 class="heading-medium">Session cookie</h2>
```
Comment (markyjones) [inline source/cookies.html.erb]: The session cookie for frontend also tracks multiple payment id's to allow multiple payments in different tabs I think.

Comment (jonheslop) [inline source/cookies.html.erb]: @markyjones ok, Tim research the cookie details but he's on holiday. How do we find out for sure?

Comment (markyjones) [inline source/cookies.html.erb]: @mrlumbu @kakumara should be able to give you the background. We do actually do this. Just not entirely sure how best to word it.

Comment (kakumara) [inline source/cookies.html.erb]: @timmacavoy / @markyjones here are the cookies details from each app.
Frontend
| Cookie name                    | description | age| 
| frontend_state                  | (obsolete) This was our old front session cookie during a payment before rotating csrf secrets | 1.5 hours |
| frontend_state_2                 | same as bove, just the newer one | 1.5 hours |
| seen_cookie_message     | as name describes |  4 weeks|

SelfService
| Cookie name                    | description | age| 
| session                          |stores the logged in state of a selfservcie user/administrator|  3 hours
| gateway_account        | stores the current selected gateway account Id (in settings area) | 30 Days 
| register_invite              | stores the state of a registration process of a user (from invitation) prior to login | 1 Hour

Comment (timmacavoy) [inline source/cookies.html.erb]: I've updated the content on the cookie policy page to reflect the relevant cookies (payment pages do not need to be covered) in this ticket  PP-2434

**Pull request merged**